Workiva/w-mobile-kit
# MOBILE-1662: Picker Action Sheet

## Description

Modify WActionSheetVC to have a base class to inherit from. Creates a new ActionSheet with a Picker instead of a table view with the optional delegates to allow interaction with Cancel and Done buttons.
## What Was Changed
- Adds an initializer with (isPickerView:Bool) and UI setup functions to display actionsheet with a pickerView and no tableview components.
- Adds an example button and view to the sample app for the pickerView
## Testing
- Launch the sample app to the ModalViewExamples page, and play around with the pickerView. 
- Ensure that the cancel and done buttons work as expected. The done button's delegate method should return the selected index at the time the button is pressed.

---

Please Review: @Workiva/mobile
